docs/products/kafka/reference/advanced-params.rst

```diff
@@ -0,0 +1,6 @@
+List of advanced parameters
+============================
+
+When creating or working with an Aiven for Apache Kafka service, several configuration options are available. They are summarized below:
```

Comment:
Should we add a mention on how to use these settings? By adding the `-c` configuration flag during Service creation (https://developer.aiven.io/docs/tools/cli/service.html#avn-service-create) or update (https://developer.aiven.io/docs/tools/cli/service.html#avn-service-update)

Reply:
My feeling was that the list was too long, it's also in ALL the tools, the API, terraform, etc. However if we did ever build something like a CLI cheat sheet for a product, we could signpost that from this page. This is reference, so likely that other pages would link to here, if that makes sense.